lib/features.js

```diff
@@ -0,0 +1,105 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict'
+
+const hash = require('sha1')
```

Comment:
IME "sha1" often triggers people's security rader, it may be worth an explicit comment stating that we're not using this for cryptographic purposes, so it's fine for us to use the (faster) sha1 here instead of sha256.  (In the future we might even consider a direct and more efficient hash-this-to-an-integer function like siphash, but sha1 will fine for now)